src/app/feed/feed-card/feed-card.component.scss

```diff
@@ -108,8 +108,9 @@ a.post-link {
 	background-color: rgba(0, 0, 0, 0.5);
 
 	.overlay-content{
+		max-width: 90%;
+		height: 340px !important;
```

Comment:
I tried to keep it at default but this made the like retweet button all merge together. At 320px, these buttons were still conflicting with Galaxy S5 model in chrome developer tools. So that's why I set it to 340px.